src/plugins/platformExternals.ts

```diff
@@ -41,7 +51,8 @@ const device = [
   'system',
   'user-activity',
   'user-profile',
-].concat(common);
+  ...common
+].concat(extraDevice ? extraDevice : []);
```

Comment:
You can do `...extraDevice` just like `...common`, if it's undefined it's no-op, if it's array (empty or not) it'll append.

Reply:
You are right. Made the change.